docs/how-to-guides/faq.md

```diff
@@ -84,6 +84,22 @@ Currently, promptflow supports the following environment variables:
 
 Valid for batch run only. The number of workers to use for parallel execution of the Flow.
 
+Default value is 16. If you want more efficiency, you can increase the number to the improve the batch run concurrency, make it run faster.
```

Comment:
Maybe it can changed to:
The default value of 'PF_WORKER_COUNT' is 16.  If you want more efficiency, you can increase the number to improve the batch run concurrency, which will make it run faster.

Note: 
If your batch run only has two lines, but "PF_WORKER_COUNT " is set to 20, it may run slower because the process startup and shutdown may take some time.

Reply:
Yes, I can add this. The idea is if the batch run data row count better be bigger than concurrency.